bundle/src/main/java/com/adobe/acs/commons/wcm/impl/DynamicClassicUiClientLibraryServlet.java

```diff
@@ -83,7 +87,7 @@ protected void doGet(@Nonnull SlingHttpServletRequest request, @Nonnull SlingHtt
         if (!excludeAll) {
             Collection<ClientLibrary> libraries = htmlLibraryManager.getLibraries(categories, LibraryType.JS, true, true);
             for (ClientLibrary library : libraries) {
-                writer.value(resourceResolver.map(library.getIncludePath(LibraryType.JS, htmlLibraryManager.isMinifyEnabled())));
+                writer.value(externalizer.relativeLink(request, resourceResolver.map(library.getIncludePath(LibraryType.JS, htmlLibraryManager.isMinifyEnabled()))));
```

Comment:
According to the JavaDoc, this is equivalent to passing the request into the `map()` method. Would you mind rewriting this as 

```
writer.value(resourceResolver.map(request, library.getIncludePath(LibraryType.JS, htmlLibraryManager.isMinifyEnabled())));
```
(and the same on the line below?)

Reply:
@justinedelson Thanks for reviewing. I updated it like requested.